### PR TITLE
changes related to nDPI-3.4(stable)

### PR DESCRIPTION
**Pull request by subhajit-cdot**
Signed-off-by: Subhajit Chatterjee <subhajit.chatterjee@cdot.in>

1. Didn't put any comment on debian/changelog

2. The code will only be compiled  with latest ndpi repository which has nDPI-3.4 support. Therefore merge ndpi repository first before taking this PR.

### CODE REVIEW

**Review by pjaitken — APPROVED** [review]

**Comment by pjaitken** [comment]
Looks good - but wait for the [nDPI update](https://github.com/danos/ndpi/pull/1) to be merged first.

**Comment by scott-leishman** [comment]
This work has been done and is now in vyatta-dataplane. I'll leave it to the dataplane maintainers to close this PR.

**Comment by nickbroon** [comment]
Changes included in 705f8aa0b0b0a7a5405bc218fc90eb29b801bbe7

**Pull request closed without merging**
